adoc/admin-centralized-logging.adoc

```diff
@@ -1,6 +1,167 @@
 = Centralized Logging
 
+Centralized Logging is a means of collecting logs from CaaS Platform for centralized management.
+It forwards system and Kubernetes cluster logs to a specified external logging service,
+for example, Rsyslog server.
 
-Centralized Logging is a means of collecting logs from CaaS Platform for centralized management. It forwards system and Kubernetes cluster logs to a specified external logging service, for example, Rsyslog server.
+Collecting logs in a central location can serve for audit or debug purposes or to analyze and visually present data.
 
-Collecting logs in a central location can serve for audit or debug purposes or to analyze and visual present the data.
+== Types of logs
+
+You can log the following groups of services, see the Usage section
+
+Kubernetes System Components::
+* Kubelet
+* Cri-o
+
+Kubernetes Control Plane Components::
+* API Server
+* Controller Manager
+* Scheduler
+* Cilium
+* Kube-proxy
+* all resources in the kube-system namespaces
+
+Kubernetes Namespaces Pods::
+* All namespaces in cluster except kube-system
+
+OS Components::
+* Kernel
+* Audit
+* Zypper
+* Network (wicked)
+
+Centralized logging is also restricted to the following protocols: UDP, TCP, TCP + TLS, TCP + mTLS.
+
+
+== Queuing
+
+Centralized Logging supports a configurable buffered queue.
+This can be used to improve log processing throughput and eliminate possible data loss,
+for instance after log agents shutdown, restart or in case of an unresponsive remote server.
+
+//TODO: how is this queue configured?
+
```

Comment:
Buffered queue can be enable/disable with following parameter:

* `*queue.enabled*`, default value = false

And additional parameters to define queue size and its disk usage:

* `*queue.size*`, default value =  50000
* `*queue.maxDiskSpace*`, default value = 2147483648

Reply:
More detailed description would be:

When queue `disabled` sets data store in-memory only.
When queue `enabled` sets  data store in mixture of in-memory and in-disk. Data stores in memory until queue filled up, then switch to disk. When enabled, it also saves queue to disk at service shutdown.

Configure queue.size sets amount of messages allows for in-memory queue. This setting effects on kubernetes cluster logs (kubernetes-control-plane and kubernetes-user-name-space).           

Configure queue.maxDiskSpace set maximum size allowed for disk storage. The storage will be partitioned 20 percent for journal logs and 80 percent for other logs.

Reply:
Thanks for the details, I will also include those. 

> Configure `queue.maxDiskSpace` set maximum size allowed for disk storage. 
In what units is this? :)

Reply:
Bytes :)